publishing-rules.yaml

```diff
@@ -2,10 +2,14 @@ skip-godeps: true
 rules:
 - destination: kubernetes
   branches:
-  - name: release-1.9.1
+  - name: origin-3.10-kubernetes-1.10.2
     source:
       branch: master
       dir: vendor/k8s.io/kubernetes
+  - name: release-1.9.1
+    source:
+      branch: release-3.9
```

Comment:
this was bugged before or something?

Reply:
Yes.  I don't think the bot is actually using this version at this point (it must not be), but while I was in here…